README.md

```diff
@@ -90,3 +90,7 @@ library("loupeR")
 # print extra debug information
 create_bugreport_from_seurat(seurat_obj)
 ```
+
+## Tutorials
+
+An [interactive script]() with an example dataset can be run from this notebook here: 
```

Comment:
You need to update the link to `misc/tutorials/5k_mouse_brain.ipynb`.  Also, I would update this to be a markdown list.  Something like:

```markdown
- [mouse brain notebook](misc/tutorials/5k_mouse_brain.ipynb): basic processing of a 10x dataset with export to a Loupe file using `loupeR`
```

Reply:
Done.